### ISSUE TITLE
训练后的结果模块保存在哪？

### ISSUE THREAD

**Issue by jiandam55** [ssbuild/chatglm_finetuning#84]
Epoch 1: 100%|██████████| 159463/159463 [29:54:30<00:00,  1.48it/s, loss=8.22, v_num=4]
Epoch 1: 100%|██████████| 159463/159463 [29:54:30<00:00,  1.48it/s, loss=8.22, v_num=4]
Epoch 1: 100%|██████████| 159463/159463 [29:54:30<00:00,  1.48it/s, loss=8.22, v_num=4]INFO:pytorch_lightning.utilities.rank_zero:`Trainer.fit` stopped: `max_epochs=2` reached.

Epoch 1: 100%|██████████| 159463/159463 [29:54:59<00:00,  1.48it/s, loss=8.22, v_num=4]

我训练了2M json数据，loss没有出现nan，最后几行日志如上，应该成功了吧。
但结果保存到哪了？
预训练模块目录里也没有？整个项目文件夹，除了日志，也没见到有其它文件变动，那末，训练后的结果模块保存到哪了呢？

**Comment by ssbuild** [comment]
lora模式在best_ckpt , last_ckpt ,  finetuning模式在 best_ckpt

**Comment by jiandam55** [comment]
喔，感谢，那这训练后的结果如何应用上？是否要改什么配置
我现在跑python web_demo.py
发现回答的结果和训练前没有明显区别，不知是训练量太少（2M）还是要改什么配置才能应用上训练结果

**Comment by ssbuild** [comment]
参考下readm,md 推理环节

**Comment by jiandam55** [comment]
推理我运行的是：python infer.py
这个具体怎么让web_demo.py应用上chatglm_finetuning/best_ckpt 的训练结果？还是不明白

**Comment by nieallen** [comment]
> lora模式在best_ckpt , last_ckpt , finetuning模式在 best_ckpt

请问lora模式想要保存每个epoch的模型，应该改什么参数？
